src/services/ui/src/components/Opensearch/main/Filtering/useFilterDrawer.ts

```diff
@@ -66,22 +80,17 @@ export const useFilterDrawer = () => {
             return updateFilter.value;
           }
           if (VAL.type === "terms") return [] as string[];
+          if (VAL.type === "match") return null;
           return { gte: undefined, lte: undefined } as opensearch.RangeValue;
         })();
 
         STATE[KEY] = { ...VAL, value };
-
         return STATE;
       }, {} as any);
     });
     setAccordionValues(updateAccordions);
   }, [url.state.filters, drawerOpen]);
 
-  // change base filters per tab
-  useEffect(() => {
-    setFilters(Consts.FILTER_GROUPS(user, url.state.tab));
-  }, [url.state.tab]);
-
```

Comment:
This was wiping out hash filters on the initial render.  
Removing this still works with tab change.